README.md

```diff
@@ -2,46 +2,31 @@
 
 JWT Authentication bundle for Symfony
 
-## Important
+## Requirements
 
-v2.x.x provides compatibility with Symfony 3
+- [PHP](http://php.net/) 5.5+
+- [Symfony](https://symfony.com/) 2.8+
+- [Auth0 PHP](https://github.com/auth0/auth0-PHP) 5.0+
 
-for Symfony 2.x, stick to v1 branch
-
-##Demo
-
-Check the usage demo: https://github.com/auth0/jwt-auth-bundle/tree/master/example
+> For Symfony < 2.8 please see [v1](https://github.com/auth0/jwt-auth-bundle/tree/1.x.x-dev)
 
 ## Installation
 
-Check our docs page to get a complete guide on how to install it in an existing project or download a pre configured seedproject: https://auth0.com/docs/quickstart/backend/php-symfony/
```

Comment:
we should check why this quickstart is not online and link to the correct document